animations/lib/src/misc/curved_animation.dart

```diff
@@ -0,0 +1,146 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'dart:math' as math;
+
+class CurvedAnimationDemo extends StatefulWidget {
+  static const String routeName = '/misc/curved_animation';
+  @override
+  _CurvedAnimationDemoState createState() => _CurvedAnimationDemoState();
+}
+
+class MyCurve {
+  Curve curve;
+  String curveName;
+  MyCurve({this.curve, this.curveName});
+}
+
+class _CurvedAnimationDemoState extends State<CurvedAnimationDemo>
+    with SingleTickerProviderStateMixin {
+  static const String routeName = '/misc/curved_animation';
+  AnimationController animController;
```

Comment:
Wherever possible in the samples repo, we try to avoid abbreviations as a way to make code more readable. Consider renaming this just to `controller` (or `animationController` if you prefer it that way).